client/src/pages/addNewSitSpot/index.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { StepsQuestions } from '../../components';
+
+import Questions from './fakeData';
+
+import './index.css';
+
+const Classes = [
+  'name',
+  'website',
+  'country',
+  'city',
+  'type',
+  'upload-img',
+  'upload-img',
+];
+
+class AddNewSitSpot extends Component {
+  state = {
+    data: {
+      name: '',
+      linkSite: '',
+      country: '',
+      city: '',
+      businessType: '',
+      imgUrlOne: null,
+      imgUrlTwo: null,
+    },
+    current: 0,
+  };
+
+  handelChange = (value, dataKey) => {
+    this.setState(prevState => ({
+      ...prevState,
+      data: { ...prevState.data, [dataKey]: value },
+    }));
+  };
+
+  next = () => {
+    const { current } = this.state;
+    this.setState({ current: current + 1 });
+  };
+
+  prev = () => {
+    const { current } = this.state;
+    this.setState({ current: current - 1 });
+  };
+
+  render() {
+    const { current, data } = this.state;
+    return (
+      <div id="add-place" className="add-place">
```

Comment:
I think you can remove the `id` and then in your stylesheet, have all of your selectors use `.add-place` instead of `#add-place`.
I try to avoid using IDs because with ID, you are limited to having only one instance of that element on the page at a time.

Reply:
yes,
- but I used it to can override on antd directly which I use only id name then antd's class.
- avoiding add many class selector before wanted class.

please, tell me what's your opinion about that :)

Reply:
I think you can still do both of those things with `.add-place` instead of `#add-place`.  You would be replacing the ID with a className, so it's still the same number of items in the selector before the wanted class.

Reply:
But also, it's not a critical change to make, just a suggestion (for next time, if you want) 👍